src/commands.ts

```diff
@@ -161,7 +164,7 @@ export function getAllByName(names: string[]): Chainable {
 export function evalMacros(
   input: Evaluatable,
   options: EvalOptions = {}
-): Chainable {
+): Cypress.Chainable {
```

Comment:
I'd advise copying another trick from RGC, which is, we can "open up" Cypress's built in type and add our own new command, which will let people use autocomplete and get popup API docs about it when they are writing tests.

Something approximately like:
```
declare global {
  /* eslint-disable-next-line @typescript-eslint/no-namespace */
  namespace Cypress {
    interface Chainable<Subject = any> {
      evalMacros: (...) => Chainable<Subject>;
    }
  }

```
but of course we'd want to copy the method signature from the real `evalMacros`, above.